### PR TITLE
Add a test scenario for default node slector

### PR DESCRIPTION
**Pull request by wanghaoran1988**
Add a test scenario for default node slector

### CODE REVIEW

**Comment by wanghaoran1988** [comment]
@derekwaynecarr Could you please help take a look when you can ?

File: plugin/pkg/admission/podnodeselector/admission_test.go [hunk 1]
```diff
@@ -141,6 +141,14 @@ func TestPodAdmission(t *testing.T) {
 			admit:                 false,
 			testName:              "Merged pod node selectors conflict with the whitelist",
 		},
+		{
+			defaultNodeSelector:             "env=dev",
+			ignoreTestNamespaceNodeSelector: true,
+			whitelist:                       "env=prd",
+			podNodeSelector:                 map[string]string{},
+			admit:                           false,
+			testName:                        "Default node slector conflict with the whitelist",
```
Comment (derekwaynecarr) [inline plugin/pkg/admission/podnodeselector/admission_test.go]: nit: typo on selector

**Review by derekwaynecarr — APPROVED** [review]

**Comment by derekwaynecarr** [comment]
@k8s-bot ok to test

**Comment by derekwaynecarr** [comment]
if you have a chance to fix the typo in the test name, it is appreciated.

this can wait to merge until the 1.7 milestone is opened up.

thanks for the PR!

File: plugin/pkg/admission/podnodeselector/admission_test.go (hunk 1 above)
Comment (wanghaoran1988) [inline plugin/pkg/admission/podnodeselector/admission_test.go]: @derekwaynecarr Thanks, updated

**Comment by wanghaoran1988** [comment]
@k8s-bot kops aws e2e test this

**Comment by wanghaoran1988** [comment]
@derekwaynecarr Comments addressed, would you mind add LGTM label, thanks.

**Pull request merged**